src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/participationhistory/restapi/CourseParticipationIntegrationTest.kt

```diff
@@ -163,35 +161,41 @@ constructor(
   fun `Updating a course participation should return 200 with correct body`() {
     val startTime = LocalDateTime.now()
     val courseId = getFirstCourseId()
-    val courseParticipationId = createCourseParticipation(minimalCourseParticipation(courseId, "A1234AA")).id
-    val updatedSource = "Source of information"
-    val updatedDetail = "Updated course participation detail"
 
-    val courseParticipationFromUpdate = updateCourseParticipation(
-      courseParticipationId,
+    val created = createCourseParticipation(
+      CreateCourseParticipation(
+        courseId = courseId,
+        prisonNumber = "A1234AA",
+      )
+    )
+
+    val updated = updateCourseParticipation(
+      created.id,
       CourseParticipationUpdate(
+        courseName = "Course name",
         courseId = courseId,
         setting = CourseParticipationSetting(
           type = CourseParticipationSettingType.custody,
         ),
-        source = updatedSource,
-        detail = updatedDetail,
+        source = "Source of information",
```

Comment:
What's the reason behind no longer storing this in a variable? Is it to be more explicit about the expected value?

Reply:
More because we were only using it in two places - the `updated` and the `expected`.
I'm rewriting both this (and the similar test for create) to more explicitly check that the results from the db match what we expect, so this might become a non-issue shortly anyway :)